### PR TITLE
INFRA - remove deprecation warnings and avoid running play if already upgraded

### PR DESCRIPTION
**Pull request by chicofranchico**

### CODE REVIEW

File: tasks/upgrade_journalnode.yml [hunk 1]
```diff
@@ -4,5 +4,5 @@
     service: name=hdfs-journalnode state=restarted
   when:
     - inventory_hostname in hdfs_journalnodes
-    - inventory_hostname == item
+    - inventory_hostname == hdfs_journalnode
```
Comment (lhoss) [inline tasks/upgrade_journalnode.yml]: good fix !
and remove the 1. and condition (implicitly contained in the narrower new condition)

File: tasks/upgrade_namenode.yml [hunk 2]
```diff
@@ -7,5 +7,6 @@
     service: name=hdfs-namenode state=restarted
   when:
     - inventory_hostname in hdfs_namenodes
-    - inventory_hostname == item
+    - inventory_hostname == hdfs_namenode
```
Comment (lhoss) [inline tasks/upgrade_namenode.yml]: idem: remove first `and` condition

File: tasks/upgrade_journalnode.yml (hunk 1 above)
Comment (chicofranchico) [reply tasks/upgrade_journalnode.yml]: good catch

**Comment by chicofranchico** [comment]
fixed

**Pull request merged**